src/css/override_vars.scss

```diff
@@ -57,6 +57,11 @@ $font-sans:                     '18Franklin-webfont', system, -apple-system, Bli
 $font-serif:                    'Georgia', 'Cambria', 'Times New Roman', 'Times', serif;
 $font-mono:                     'Roboto Mono', 'Bitstream Vera Sans Mono', 'Consolas', 'Courier', monospace;
 
+// utility class scale suffixes
+
+$u-breakpoints: '-xs', '-s', '-m', '-l', '-xl';
+$u-flexscale: '-f1', '-f2', '-f3', '-f4', '-f5', '-f6', '-f7', '-f8', '-f9', '-f10', '-f11';
```

Comment:
Seems there's a compromise between succinctness and readability — in the world of utility classes, I tend to want to err in the direction of succinctness.

There are [n] scales in the utility classes:

grid scale: [#]
type scale: s[#]
flex scale: f[#]
pixel scale: [#]px
percentage scale: [#]ct 

To me, this feels right.

However, if you want the actual class name in the standard scss to be more like `.col-flex-1` — that seems ok.

Actually, this `$u-flexscale` var is only used in one place — `utility-classes.scss:420` and that could be rewritten...